### DISCUSSION TITLE
Variable Scaling in Highs

### DISCUSSION THREAD

**Discussion by DillonJ** [ERGO-Code/HiGHS#1237] [Ideas]
Dear HiGHS team,

Thanks everyone again for creating a very viable open-source MIP solver alternative. We're using it quite extensively over at the Spine, open source energy system modelling framework project. (https://github.com/spine-tools).

One of my colleagues has had the experience that scaling of variables seems to have an order of magnitude impact on HiGHS performance. This seems to be something that commercial solvers do automatically. Perhaps this is something that could be considered for HiGHS or perhaps reviewed if the functionality already exists?

Here is the original issue that has a little more info: https://github.com/spine-tools/SpineOpt.jl/issues/585

**Comment by jajhall** [comment]
HiGHS scales by default, but the effectiveness of different scaling techniques is problem (class) dependent. So it's quite possible that something bespoke - or the technique in commercial codes - does better. I'll check out the discussion on spine.
